src/qt/bitcoinstrings.cpp

```diff
@@ -157,6 +157,9 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 QT_TRANSLATE_NOOP("bitcoin-core", ""
 "The transaction amount is too small to send after the fee has been deducted"),
 QT_TRANSLATE_NOOP("bitcoin-core", ""
+"This is a pre-release test build - use at your own risk - do not use for "
```

Comment:
Strange, this one is still missing - You might have to run `make translate` again to make sure these end up in bitcoin_en.ts as well.